### PR TITLE
Little updates to make life easiar

### PR DESCRIPTION
**Pull request by natan515**
No more of those anoying refresh messages when u open ur GUI with HT on, and it tells players if they are not in the right Y LVL for the bot.

### CODE REVIEW

**Comment by natan515** [comment]
Tmr, il do some more changes, like that it would be just in singlaplayer (the gui thing)

**Comment by natan515** [comment]
Ok, now it's done. You can look at this and decide if u want to add it. (;

**Comment by natan515** [comment]
Just changed blueprint to bluePrint like you normaly should write in Kotlin.

File: src/main/java/me/zeroeightsix/kami/module/modules/misc/HighwayTools.kt [hunk 1]
```diff
@@ -911,7 +912,7 @@ object HighwayTools : Module() {
                         append("\n    §9> §7Coordinate: §a${startingBlockPos.z}§r")
                     }
                 }
-
+                if (startingBlockPos.y >= 118 && startingBlockPos.x < 120) append("\n    §9> §7You should move to Y 120 to build proper highways")
```
Comment (theredstoner) [inline src/main/java/me/zeroeightsix/kami/module/modules/misc/HighwayTools.kt:915]: The second condition should be .y not .x, also you may want to replace the check with:
`if (startingBlockPos.y != 119)` (or whatever the correct block height is)

Comment (natan515) [reply src/main/java/me/zeroeightsix/kami/module/modules/misc/HighwayTools.kt:915]: Ur right, ima change thanks for showing me my mistake 👍

Comment (natan515) [reply src/main/java/me/zeroeightsix/kami/module/modules/misc/HighwayTools.kt:915]: The thing is that you do use different Y sometimes to build highways so I don't want to do it !- 120, cus there is no reason for you to build highways at Y 118-119 but there is for 117 and lower.

**Comment by natan515** [comment]
Ok, I changed it to 117 it's done now.

**Comment by natan515** [comment]
@Avanatiker Can you please Merge it? I mean only if you think it's ok.

**Comment by Avanatiker** [comment]
Thank u a lot :) nice idea with the checks. blueprint is one word btw :P

**Pull request merged**